go/kbfs/libkbfs/config_local.go

```diff
@@ -1518,6 +1527,8 @@ func (c *ConfigLocal) loadSyncedTlfsLocked() (err error) {
 			// case, since the cleaning behavior depends on it being
 			// nil if there has been an error.
 			c.syncedTlfs = nil
+			c.GetPerfLog().CDebugf(
+				context.TODO(), "Failed to open database: %v", err)
```

Comment:
can you prefix this to indicate it's KBFS related?

Reply:
Good idea, done!